monkey/monkey_island/cc/services/aws/aws_command_runner.py

```diff
@@ -6,10 +6,18 @@
 import botocore
 from egg_timer import EggTimer
 
+from common.common_consts import AGENT_OTP_ENVIRONMENT_VARIABLE
+from common.types import OTP
+
 STATUS_CHECK_SLEEP_TIME = 1
 LINUX_DOCUMENT_NAME = "AWS-RunShellScript"
 WINDOWS_DOCUMENT_NAME = "AWS-RunPowerShellScript"
 
+# Setting the log level of botocore to CRITICAL so that
+# it doesn't log out the commands sent to the AWS instance,
+# thus hiding the Agent's OTP
+logging.getLogger("botocore").setLevel(logging.CRITICAL)
+
```

Comment:
Isn't this already handled by OTPFormatter?

Reply:
For some reason it wasn't.